### PR TITLE
Common components

### PR DESCRIPTION
**Pull request by thelittlecitizen18**
navbar & tabbar

### CODE REVIEW

File: src/Components/CommonComponents/Tabs/Test1.jsx [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+          Item Two
+        </TabPanel>
+        <TabPanel value={value} index={2} dir={theme.direction}>
+          Item Three
+        </TabPanel>
+      </SwipeableViews>
+    </div>
+  );
+}
+import React from 'react';
```
Comment (thelittlecitizen19) [inline src/Components/CommonComponents/Tabs/Test1.jsx:98]: move all imports to the top of the page

File: src/Components/CommonComponents/Tabs/Test1.jsx [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    borderRadius: 3,
+    border: 0,
+    color: 'white',
+    height: 48,
+    padding: '0 30px',
+    fontSize: 500
+  },
+};
+
+function ClassNames(props) {
```
Comment (thelittlecitizen19) [inline src/Components/CommonComponents/Tabs/Test1.jsx:235]: change the name to more indicative name

File: src/Components/CommonComponents/Tabs/Test1.jsx [hunk 3]
```diff
@@ -0,0 +1,255 @@
+import React from 'react';
```
Comment (thelittlecitizen19) [inline src/Components/CommonComponents/Tabs/Test1.jsx:1]: seperate components between files

File: src/Components/CommonComponents/Tabs/Test1.jsx [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.background.paper,
+    width: 500,
+  },
+}));
+
+export default function FullWidthTabs() {
```
Comment (thelittlecitizen19) [inline src/Components/CommonComponents/Tabs/Test1.jsx:51]: use only one export default in file, you can use export with { } to export multipal objects by name

File: src/Components/CommonComponents/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import './Navbar.css'
+
+
+
+function Navbar({navbarList}) {
+    return (
+        <div>
+            <nav className="navbar navbar-expand-lg navbar-light">
+                <a className="navbar-brand" href="#">Feldmanager</a>
```
Comment (thelittlecitizen19) [inline src/Components/CommonComponents/Navbar/Navbar.jsx:10]: get feldmanager string as props property

**Review by thelittlecitizen19 — CHANGES_REQUESTED** [review]

File: src/Components/CommonComponents/Navbar/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+                <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                    <span className="navbar-toggler-icon"></span>
+                </button>
+                <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
+                    <div className="navbar-nav">
+
+                        {navbarList.map((item)=>{
```
[the remark this answers is not in the document]
Comment (thelittlecitizen18) [reply src/Components/CommonComponents/Navbar/Navbar.jsx:17]: done

File: src/Components/CommonComponents/Navbar/Navbar.jsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+                            return(
```
[the remark this answers is not in the document]
Comment (thelittlecitizen18) [reply src/Components/CommonComponents/Navbar/Navbar.jsx:18]: done

**Review by thelittlecitizen19 — APPROVED** [review]

**Pull request closed without merging**